code/index.html

```diff
@@ -1,22 +1,75 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+  <!-- HEADER -->
+  <header>
+    <div class="logo-container">
+      <img class="logo" src="./logo.svg">
+    </div>
+    <div class="header-heading"><h1 class="heading">Swecation Magazine</h1></div>
+  </header>
+  <nav>
+    <a href="#">North</a>
+    <a href="#">East</a>
+    <a href="#">South</a>
+    <a href="#">West</a>
+  </nav>
+
+  <!-- ARTICLE -->
+  <article class="news-container">
+    <div class="news">
+      <img src="salt.JPG" class="news-img" alt="Sea waves and cliffs">
+      <p class="tag">West edition</p>
+      <h2>Island hopping</h2>
+      <p>It’s easy to get out to the thousands of islands in the Bohuslän archipelago with the help of local ferry and boat companies. You’ll find tips here on how to find your own favourite spot on the west coast – from the Kosterhavet sea in the north, to Gothenburg’s archipelago in the south.
+        <br>A number of different boat companies operate along the coast of Bohuslän. We’ve gathered them all here with the information you need to be able to plan your trip. 
+      </p>
+      <a href="#" a class="button">Read more</a>
+    </div>
+    <div class="news">
+      <img src="crayfishsquared.JPG" class="news-img" alt="Crayfish on a table">
+      <p class="tag">West edition</p>
+      <h2>Seafood safari</h2>
+      <p>Outside the summer season you find the genuine Bohuslän - salty, barren, windswept and calm. Head out on a shellfish tour, to catch the finest prawns, mussels, oysters, langoustines and lobster. Add a little salt to your life.
+        <br>
```

Comment:
Instead if using <br> try to find a way of styling it the way you'd like or using another <p> instead 😎 Maybe next project?